http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java

```diff
@@ -14,10 +14,8 @@
 @QuarkusScenario
 public class HttpAdvancedReactiveIT extends BaseHttpAdvancedReactiveIT {
 
-    //TODO Remove workaround after Keycloak is fixed https://github.com/keycloak/keycloak/issues/9916
     @KeycloakContainer(command = { "start-dev", "--import-realm", "--hostname-strict=false" })
-    static KeycloakService keycloak = new KeycloakService(DEFAULT_REALM_FILE, DEFAULT_REALM, DEFAULT_REALM_BASE_PATH)
-            .withProperty("JAVA_OPTS", "-Dcom.redhat.fips=false");
+    static KeycloakService keycloak = new KeycloakService(DEFAULT_REALM_FILE, DEFAULT_REALM, DEFAULT_REALM_BASE_PATH);
```

Comment:
Yeah, it is fixed for a long time, but I never got to fix it on our side. If you really want to drop this flag, we need green FIPS run. I have vague memory it didn't work when I dropped it and it required more work, but it's a _long_ time ago. Things could have changed. Please run it on FIPS and confirm back.

Reply:
BTW it is nice to have; we need to know that Quarkus side works on FIPS enabled env, not the integrations. But there is definitely work on Keycloak / mTLS Keycloak tests that deservers to be done when we have a time.

Reply:
If you can check `io.quarkus.ts.security.oidcclient.mtls.Pkcs12OidcMtlsIT` I'd appreciate it. The difference is in secured communication.

Anyway, as  you already checked the HTTP module, it means that we can remove it at least somewhere. Thanks

Reply:
These are the results for `mvn clean verify -pl security/oidc-client-mutual-tls/`:
```
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 27.21 s -- in io.quarkus.ts.security.oidcclient.mtls.Pkcs12OidcMtlsIT
```

Reply:
Thank you